Assets/Scripts/Auction/BiddingPlatform.cs

```diff
@@ -95,7 +95,7 @@ public bool TryPlaceBid(PlayerIdentity playerIdentity)
             return false;
         }
 
-        if (playerIdentity.chips > chips)
+        if (playerIdentity.chips > chips || (playerIdentity == leadingBidder && playerIdentity.chips > 0))
```

Comment:
You may want to extract these two cases into variables for clarity

Reply:
Fixed!